components/renku_data_services/users/apispec.py

```diff
@@ -22,6 +23,11 @@ class Version(BaseAPISpec):
     version: str
 
 
+class SecretKind(Enum):
+    general = "general"
+    storage = "storage"
```

Comment:
In my experience, enums are usually in all caps:

```suggestion
    GENERAL = "GENERAL"
    STORAGE = "STORAGE"
```

Reply:
Clarified -- in this repo lowercase is used for enums.